xml/deploy_cephadm.xml

```diff
@@ -701,131 +701,293 @@ o- time_server ................................................ [enabled]
+       </substeps>
+      </step>
+     </procedure>
+     <procedure>
+      <title>Configure the local registry and access credentials</title>
+      <step>
+       <para>
+        Configure the URL of the local registry:
+       </para>
+<screen>&prompt.cephuser;ceph-salt config /containers/registry_auth/registry set <replaceable>REG_HOST_URL</replaceable></screen>
+      </step>
+      <step>
+       <para>
+        Configure the user name and password to access the local registry:
+       </para>
+<screen>&prompt.cephuser;ceph-salt config /containers/registry_auth/username set <replaceable>REG_USERNAME</replaceable></screen>
+<screen>&prompt.cephuser;ceph-salt config /containers/registry_auth/password set <replaceable>REG_PASSWORD</replaceable></screen>
+      </step>
+      <step>
+       <para>
+        Run <command>ceph-salt apply</command> to update the &salt; pillar on
+        all minions.
+       </para>
+      </step>
+     </procedure>
+     <tip>
+      <title>Registry cache</title>
       <para>
-       Run <command>ceph-salt apply</command> to update the &salt; pillar on
-       all minions.
+       To avoid re-syncing the local registry when new updated containers
+       appear, you can configure a <emphasis>registry cache</emphasis>.
       </para>
-     </step>
-    </procedure>
-    <tip>
-     <title>Registry cache</title>
+     </tip>
      <para>
-      To avoid re-syncing the local registry when new updated containers
-      appear, you can configure a <emphasis>registry cache</emphasis>.
-<!-- 2021-02-09 tbazant: still looking for better solution
-      Find
-      more information about containers and air-gapped scenarios in
-      <link
-       xlink:href="https://documentation.suse.com/sbp/all/single-html/SBP-Private-Registry/#_deployment_in_an_air_gapped_environment"/>.
-      -->
+      Cloud Native Application Development and Delivery methods require a
+      registry and a CI/CD (Continuous Integration/Delivery) instance for the
+      development and production of container images. You can use a private
+      registry in this instance.
      </para>
-    </tip>
-    <para>
-     Cloud Native Application Development and Delivery methods require a
-     registry and a CI/CD (Continuous Integration/Delivery) instance for the
-     development and production of container images. You can use a private
-     registry in this instance.
-<!-- 2021-02-09 tbazant: still looking for better solution
-     For more information, see
-     <link
-       xlink:href="https://documentation.suse.com/sbp/all/single-html/SBP-Private-Registry/"/>.
-     -->
-    </para>
+    </sect4>
+    <sect4 xml:id="deploy-cephadm-configure-imagepath">
+     <title>Configuring the path to container images</title>
+     <tip>
+      <title>Configuring HTTPS proxy</title>
+      <para>
+       If you need to use a proxy to communicate with the container registry
+       server, perform the following configuration steps on the &adm;:
```

Comment:
I believe these steps are necessary on all nodes, not just the admin node, as all nodes need to be able to access the container registry.